frb_codegen/src/method_utils.rs

```diff
@@ -0,0 +1,181 @@
+use crate::ir::IrType::{self, StructRef};
+use crate::ir::{IrField, IrTypeStructRef};
+use crate::ir::{IrFile, IrFunc};
+use convert_case::{Case, Casing};
+
+const STATIC_METHOD_MARKER: &str = "__static_method___";
```

Comment:
nit: maybe a line of comment discussing why it is 2 underline before and 3 underline after it. readers may intuitively think it is both 2 underlines before and after.

or better, make them both 2 (or both 3?) underlines